gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -13,19 +14,50 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-# global export for unqualified use in Pods file
-def configure_for_earlgrey(*args)
-  EarlGrey.configure_for_earlgrey(*args)
+require 'colored'
+require 'fileutils'
+require 'xcodeproj'
+
+def configure_for_earlgrey
+  raise <<-S
+    configure_for_earlgrey in post_install is now deprecated, you should safely please it now.
```

Comment:
`you should safely please it now.`
?

maybe instead of immediately breaking everyone by raising an exception, we should only print a warning and then call through to `EarlGrey.configure_for_earlgrey`? Then in the next release we can drop the global method entirely.